tests/e2e/run-local.sh

```diff
@@ -69,19 +69,21 @@ main() {
 		exit 1
 	fi
 
+	export "PATH=$PATH:/usr/local/bin"
+
 	pushd "$script_dir" >/dev/null
 	echo "INFO: Bootstrap the local machine"
 	ansible-playbook -i localhost, -c local --tags untagged ansible/main.yml
 
 	echo "INFO: Bring up the test cluster"
-	sudo ./cluster/up.sh
+	sudo -E PATH="$PATH" bash -c './cluster/up.sh'
 	export KUBECONFIG=/etc/kubernetes/admin.conf
 
 	echo "INFO: Build and install the operator"
-	sudo -E PATH="$PATH" ./operator.sh
+	sudo -E PATH="$PATH" bash -c './operator.sh'
 
 	echo "INFO: Run tests"
-	cmd="sudo -E PATH=\"$PATH\" ./tests_runner.sh"
+	cmd="sudo -E PATH=\"$PATH\" bash -c './tests_runner.sh'"
```

Comment:
`'` here may lead `-r $runtimeclass` failed to append to tests_runner.sh as parameter

Reply:
Good catch! Gonna fix it!